### ISSUE TITLE
Notification message to advise account needs to be approved

### ISSUE THREAD

**Issue by zigojacko** [Vinai/customer-activation#123]
Is there meant to be an on-screen notification to advise the customer that the account will need to be approved by an admin?

I've just tested this and the login page just refreshed with no notification or message at all.

**Comment by Vinai** [comment]
Yes, see https://github.com/Vinai/customer-activation/blob/master/app/code/community/Netzarbeiter/CustomerActivation/Model/Observer.php#L53-L64

That said, I haven't used this module in years, and it might be incompatible with the latest Magento 1 versions.
I assume you are using it with Magento 1 and not Magento 2, yes?

**Comment by zigojacko** [comment]
Ahh yes, no problem, I'll look into it to see if I can figure out why it's not showing.

Yes, Magento 1.9.3.8 testing in.

Thanks.
